src/hyp3_isce2/insar_tops_burst.py

```diff
@@ -219,7 +219,6 @@ def insar_tops_single_burst(
 def insar_tops_multi_burst(
     reference: list[str],
     secondary: list[str],
-    swaths: list = [1, 2, 3],
```

Comment:
Can you give me a bit more detail on this change? Why do we remove the `swaths` argument? Was it just not being used?

Reply:
Yep, it was unused. It's set explicitly in this function:

```python
swaths = list(set(int(granule.split('_')[2][2]) for granule in reference))
```

but the parameter value is never used. Also, `insar_tops_multi_burst` is only called in once place in this repo (at the bottom of this file), and `swaths` is not passed there.

Also note that using a mutable default argument such as `[1, 2, 3]` is discouraged, so PyCharm complains about that as well: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments